pkg/controller/studyjobcontroller/model_db_util.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The Kubeflow Authors
```

Comment:
This is not only for ModelDB.
It will be an interface for the model management systems including ModelDB.
So I want to rename it to model_util.go.